src/adapter.ts

```
@@ -222,23 +215,27 @@ export class SequelizeAdapter<
     return sequelize;
   }
 
+  handleError (error: any) {
+    return errorHandler(error);
+  }
```

It's really really important for `handleError` or `errorHandler` to `throw` an error. If you overwrite `handleError` with a function that does not `throw` anything, you swallow the error easily.

We call that function in every `.catch()` clause. We just should be double sure, that the error cannot be swallowed. So at least we should add a private `_handleError` function or something that calls the overwritable `handleError` function and throws the error at the end. If `handleError` throws an error, our fallback error won't be reached.

In addition the default `errorHandler` is pretty convenient. With a custom handler it's easy to mess things up. What if `errorHandler` already receives the wrapped error from `errorHandler`? Does this fit your use case? Or is a signature `handleError(feathersError: any, sequelizeError: any)` conceivable?

---

This is good feedback. I intended to use it like this
```js
handleError(error) {
  if (error.message regex something) {
    error.message = "Safe Error message"
  }
  return super.handleError(error);
}
```
But you are right. Other users may not handle and re-throw the error and it would be swallowed. I like the idea of wrapping our use in another handler. No need to attach it to the class as `_handleError`. We will just keep a private function and wrap our own usage of `handleError`.

---

@fratzinger This has been updated with your suggestions. The double `try/catch` thing in `catchHandler` is a little goofy. So I stared to update `errorHandler` to be `convertError` and return an error instead of throw one. But then I realized `errorHandler` is exported from the package and is probably more useful as is than as a `convertError`. So even though the try/catch thing is a little goofy, I think its fine.

---

I also used `GeneralError` instead of `Error` in the few other places.